### PR TITLE
fix: css & description fix for create modal

### LINKED ISSUE #6611: [3/29 bugbash] Incorrect copy in open dialog
from Em Ivers https://fuselabs.visualstudio.com/BotFramework/_queries/edit/58621/?triage=true

The dialog for adding Azure bot resource to an existing bot has incorrect copy.

As implemented:
image

Should be:
image

"Specify an existing bot to connect to your Azure Bot resource."

## Version
v1.4.0-nightly.228616.da1ac55

### PR DESCRIPTION
**Pull request by alanlong9278**
## Description

fix: css & description fix for create modal
- width adjust.
image

- description adjust
image

## Task Item

closes #6611  #6610

### CODE REVIEW

File: Composer/packages/client/src/components/CreationFlow/v2/CreateOptions.tsx [hunk 1]
```diff
@@ -61,6 +63,27 @@ export function CreateOptionsV2(props: CreateOptionsProps) {
...
+        paddingTop: '14px',
+        paddingBottom: '11px',
+      },
+      subText: {
+        fontSize: FontSizes.size14,
+      },
+    },
+    modal: {
+      main: {
+        // maxWidth: '416px !important',
```
Comment (hatpick) [inline Composer/packages/client/src/components/CreationFlow/v2/CreateOptions.tsx:80]: remove

File: Composer/packages/client/src/constants.ts [hunk 2]
```diff
@@ -227,7 +227,7 @@ export const DialogCreationCopy = {
   get SELECT_LOCATION() {
     return {
       title: formatMessage('Select a Bot'),
-      subText: formatMessage('Which bot do you want to open?'),
+      subText: formatMessage('Specify an existing bot to connect to your Azure Bot resource.'),
```
Comment (hatpick) [inline Composer/packages/client/src/constants.ts:230]: do we want to change the text for all the scenarios? what if a user is just simply opening a bot?

**Review by hatpick — DISMISSED** [review]

File: Composer/packages/client/src/components/CreationFlow/v2/CreateOptions.tsx (hunk 1 above)
Comment (alanlong9278) [reply Composer/packages/client/src/components/CreationFlow/v2/CreateOptions.tsx:80]: Removed

File: Composer/packages/client/src/constants.ts (hunk 2 above)
Comment (alanlong9278) [reply Composer/packages/client/src/constants.ts:230]: @Andrea-Orimoto Should we change the text for all the scenarios?

Comment (cwhitten) [reply Composer/packages/client/src/constants.ts:230]: We should not change the text for all scenarios. It's important that we preserve the copy for scenarios that are not the ABS Handoff scenario

Comment (alanlong9278) [reply Composer/packages/client/src/constants.ts:236]: @cwhitten Sure, I will fix it today.

**Review by boydc2014 — APPROVED** [review]

**Pull request merged**